README.md

```diff
@@ -40,7 +40,7 @@ It supports both lambdas with code in an S3 bucket as well as those deployed as
 
   # Example entry in functions.yml for an function in a Docker container
   repo: serverless-tools
-  registry_id: 123456789
+  registry_id: "123456789"
```

Comment:
Would a better idea here be to cast registry_id to a string rather than having to quote

Reply:
actually no - scratch that, registry IDs can have leading zeros :(